raiden/network/transport/matrix/transport.py

```diff
@@ -525,6 +537,9 @@ def stop(self) -> None:
         self._stop_event.set()
         self._broadcast_event.set()
 
+        if self._raiden_service:
+            self._web_rtc_manager.stop()
```

Comment:
Can `self._raiden_service` ever be `None` here?

Reply:
There is only one place where the stop_event is cleared (upon `start()`)
directly afterwards `_raiden_service` is set. So should be fine to not ask for existence